contrib/utilities/relocate_libraries.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
```

Comment:
I imagine @tamiko will ask you to put the usual copyright and license header into this file.